PRIVACY.md

```diff
@@ -58,7 +58,7 @@ The following never leave your environment and are not sent or shared with anyon
 The following data will never leave your environment as part of metrics.
```

Comment:
This language is confusing - as we then go on to say that it will leave your environment under certain conditions

Reply:
It was confusing to me as well. I opted to not update the language outside the scope of these changes but that should maybe a rewording here would be nice. I can ping technical writers to check in on this